src/openvpn/ps.c

```diff
@@ -433,11 +431,9 @@ proxy_entry_new(struct proxy_connection **list,
     struct proxy_connection *cp;
 
     /* connect to port share server */
-#if defined(TARGET_LINUX) && defined(ENABLE_MPTCP)   
-    if ((sd_server = socket(PF_INET, SOCK_STREAM, IPPROTO_MPTCP)) < 0)
-#else
-    if ((sd_server = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP)) < 0)
-#endif
+    /* TODO: provide access to the options structure */
```

Comment:
Sorry, I don't think I can help with that. Maybe this option has to be duplicated in a different structure here (similar to `sock->info.mptcp = o->enable_mptcp;` for `socket.c`), or passed by the caller.

But before doing that, do you know when this function is used? Is it used when connecting to OpenVPN server via an HTTP proxy? Maybe MPTCP support for this can be added later to that, no?